climakitae/core/data_interface.py

```diff
@@ -0,0 +1,799 @@
+import os
+import pkg_resources
+import pandas as pd
+import geopandas as gpd
+from shapely.geometry import box, Polygon
+import intake
+import param
+import warnings
+from matplotlib.figure import Figure
+import matplotlib.ticker as ticker
+from climakitae.core.constants import (
+    variable_descriptions_csv_path,
+    stations_csv_path,
+    data_catalog_url,
+)
+from climakitae.util.utils import read_csv_file
+from climakitae.core.boundaries import Boundaries
+from climakitae.util.unit_conversions import _get_unit_conversion_options
+from climakitae.core.catalog_convert import (
+    _downscaling_method_to_activity_id,
+    _resolution_to_gridlabel,
+    _timescale_to_table_id,
+    _scenario_to_experiment_id,
+)
+
+
+class DataInterface:
```

Comment:
I will make this more abstract and not have it load the data and call it on package import I guess. Then it will load data when data is requested...